README.md

```diff
@@ -212,16 +212,23 @@ The console output is designed to make it easy to copy and paste into a spreadsh
 
 ### Jupyter Notebook
 
+#### Prerequisites
 If you want to generate graphs in a Jupyter Notebook, there is some additional setup that you need to follow.
 
-1. Install Jupyter-Lab or nteract
-1. Install dontet/interactive
+1. Install [Jupyter-Lab](https://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html) or nteract.
+1. Install [dontet/interactive.](https://github.com/dotnet/interactive/blob/main/docs/NotebooksLocalExperience.md)
 
+#### Sample Notebook
 A [sample notebook](https://github.com/corgibytes/freshli/blob/main/Sample.ipynb) can be found in this repository.
+To run the statements within this sample file, you will need to pull down a local version of
+[XPlot]() and run the build script located inside of this project. Note that, currently,
+the XPlot build will fail unless the 3.1.201 SDK is installed on your local machine as a breaking
+change to this project was introduced in dotnet SDK 3.1.300. However, for the sample below, this step is unnecessary.
```

Comment:
Never mind I just found:

https://github.com/dotnet/interactive/issues/534

It appears that Jupyter ships with it's own version of XPlot.  Lots of hoops to jump through to get the sample [sample notebook](https://github.com/corgibytes/freshli/blob/main/Sample.ipynb) to work.  Is there anything we can do to make it simpler?  Something that does not involve compiling your own version of `dotnet/interactive`.

Should we remove the custom XPlot features, like `stackgroup` form the sample for now?  Then once @mscottford submits his PR we could look at re-adding the `stackgroup` feature to the sample.

Thoughts?

Reply:
That might be a good way forward, though I wondering if we even need this sample with the other documentation provided.

Reply:
@mrbiggred Do you think that the example in the README is sufficient?

Reply:
@dan-hein I think the example in the README is a good starting point.  It is a good smoke test that you have Jupyter and Freshli setup correctly.  It would be nice to have both a simple smoke test example and a more complicated example.

The issue I have with the current [Sample](https://github.com/corgibytes/freshli/blob/main/Sample.ipynb) is that it does not work out of the box.  At least not for me.  I would like to keep this more complex example but remove the `stackgroup` option at the end that requires a customer XPlot.

```
var areaSeries = new List<Graph.Scatter>();

foreach (var name  in names) {
    areaSeries.Add(new Graph.Scatter {
        name = name,
        x = project.Select(r => r.Date),
        y = project.Where(r => r.LibYear.Any(p => p.Name == name)).Select(r => r.LibYear[name].Value),
        fill = "tonexty",
        stackgroup = "one"  # REMOVE THIS LINE
    });
}

var chart = Chart.Plot(areaSeries.ToArray());
chart.WithTitle("LibYear over time per dependency");
display(chart)
```

Finally I think it would be better if the simple smoke test example is `ipynb` file instead of copy/paste from the readme.  Thoughts?

Reply:
@dan-hein I like @mscottford's idea of setting up a Docker container but not right now.  I would like to see these ReadMe changes merged so @dfar-io can do the [v0.3.0](https://github.com/corgibytes/freshli/milestone/4) ReadMe clean up (#242).

For now I think it's best if the Jupyter instructions said something like:

1) Install Juptyer with link to install instructions and leave it up to the reader to install however they want.
2) Open the smoke test sample in Jupyter.
3) Change the link to the Freshli DLL.
4) Run the sample.

We can also have a link to the more complicated Jupyter sample but that sample should have the `stackgroup` property removed for now.  Once fslaborg/XPlot#144 is merged we can update the ReadMe.  We can also update the samples when we push Freshli to NuGet.

In summary update the documentation and samples to work with what we have now.  We will update the documentation when Jupyter/XPlot PRs are merged and when Freshli Core makes it opening debut on NuGet.

Thoughts?